### ISSUE TITLE
Uncertainty Language 

### ISSUE THREAD

**Issue by karhide-xx** [karhide-xx/SR15-Microsite#1]
Check for

confidence
likely 
unlikely 
agreement
virtually certain

evidence
	no evidence - Should not be 
	robust evidence
	limited evidence
	high evidence
	little evidence
	substantial evidence
	

Chapter 1 Done
Chapter 2 Done
Chapter 3 Done
Chapter 4 Done
Chapter 5 

Chapter 3
==============

Check 3.4.5.1

TABLE 3.1: - Duplicate text title and caption

Chapter 3 - ES - Broken Footnote
Climate models project robust differences in regional climate between present-day and global warming up to 1.5°C, and between 1.5°C and 2°C2 (high confidence), depending on the 

Chapter 3 - Section 3.3.2.2 - Missing Bookmark 'regions1'
In summary, there is high confidence that there are robust and statistically significant differences in the projected temperature means and extremes at 1.5°C versus 2°C of global warming, both in the global average and in nearly all land regions1 (likely). 

Chapter 3 - Box 3.3 - Check it is the correct image looks better in PDF

Should 'considerable and compelling evidence' be in italics

Check the reset of evidence for chapter 3

**Comment by karhide-xx** [comment]
Should robust evidence be in italics?

**Comment by karhide-xx** [comment]
This has been fixed: 
Chapter 3 - ES - Broken Footnote
Climate models project robust differences in regional climate between present-day and global warming up to 1.5°C, and between 1.5°C and 2°C2 (high confidence), depending on the

**Comment by karhide-xx** [comment]
TABLE 3.1: - Duplicate text title and caption - REMOVED

**Comment by karhide-xx** [comment]
This has now been fixed: 

Chapter 3 - Section 3.3.2.2 - Missing Bookmark 'regions1'
In summary, there is high confidence that there are robust and statistically significant differences in the projected temperature means and extremes at 1.5°C versus 2°C of global warming, both in the global average and in nearly all land regions1 (likely).
